docs/3____physical_signal_abstraction.adoc

```diff
@@ -2,7 +2,10 @@
 
 === Overview
 
-#ToDo#: Write Overview content here
+Physical Signal Abstraction allows an idealized exchange of unit-based variables between FMUs.
+In AUTOSAR, this would be equivalent to the Virtual Function Bus.
+Variables are clocked to reflect the timing aspects of the information flow.
+Changes to these clocked variables always reach the destination.
```

Comment:
I think we should do that in a bit more detail. Possibly go into the hierarchy of frames, PDUs and signals again briefly (just a  short teaser). I would also like a picture (similar to the "low cut" overview). I'm trying to make a suggestion. Can I use your branch?